tests/devapps/MSI/HelperService/MSIHelperService/Controllers/GetMSITokenController.cs

```diff
@@ -2,11 +2,36 @@
 // Licensed under the MIT License.
 
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.Identity.Test.Common.Http;
 using MSIHelperService.Helper;
 using Swashbuckle.AspNetCore.Annotations;
 
 namespace MSIHelperService.Controllers
 {
+    [ApiController]
+    [Route("[controller]")]
+    [SwaggerTag(description: "Gets MSI Token")]
+    public class HttpController : ControllerBase
+    {
+        private readonly ILogger _logger;
+        private readonly IHttpClientFactory _httpClientFactory;
+
+        public HttpController(ILogger<GetMSITokenController> logger, IHttpClientFactory httpClientFactory)
+        {
+            _logger = logger;
+            _httpClientFactory = httpClientFactory;
+        }
+
+        [HttpPost]
```

Comment:
Not sure if POST is needed, but probably safer that GET since url length can be an issue.